### PR TITLE
otp sdk

### PR DESCRIPTION
**Pull request by ahmd-alli**

### CODE REVIEW

File: mslm/client.go [hunk 1]
```diff
@@ -21,6 +22,9 @@ type Client struct {
 
 	// The Email Verify API client.
 	EmailVerify *email_verify.Client
+
+	// The Otp client.
```
Comment (UmanShahzad) [inline mslm/client.go:26]:
```suggestion
	// The OTP client.
```

File: otp/verify.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	// Prepare options.
+	opt := &OtpTokenVerifyReqOpts{}
+	if len(opts) > 0 {
+		opt = opts[len(opts)-1]
+	}
+	opt.ReqOpts = c.C.PrepareOpts(&opt.ReqOpts)
+
+	// Prepare URL.
+	qp := map[string]string{}
+	tUrl, _ := url.Parse("/api/v1/send")
```
Comment (UmanShahzad) [inline otp/verify.go:38]:
```suggestion
```

File: otp/send.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	// Prepare options.
+	opt := &OtpSendReqOpts{}
+	if len(opts) > 0 {
+		opt = opts[len(opts)-1]
+	}
+	opt.ReqOpts = c.C.PrepareOpts(&opt.ReqOpts)
+
+	// Prepare URL.
+	qp := map[string]string{}
+	tUrl, _ := url.Parse("/api/v1/send")
```
Comment (UmanShahzad) [inline otp/send.go:42]:
```suggestion
```

File: otp/send.go [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+	TmplSms       string `json:"tmpl_sms"`
+	TokenLen      int    `json:"token_len"`
+	ExpireSeconds int    `json:"expire_seconds"`
+}
+
+func (c *Client) Send(
+	otpSendReq *OtpSendReq,
+	opts ...*OtpSendReqOpts,
+) (*OtpSendResp, error) {
+	fmt.Println("here send func start")
```
Comment (UmanShahzad) [inline otp/send.go:32]:
```suggestion
```

File: otp/send.go [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+		return nil, err
+	}
+
+	// Get data.
+	otpSendResp := &OtpSendResp{}
+	if err := c.C.ReqAndResp("POST", tUrl, data, otpSendResp, &opt.ReqOpts); err != nil {
+		return nil, err
+	}
+
+	fmt.Println("here send func end")
```
Comment (UmanShahzad) [inline otp/send.go:60]:
```suggestion
```

File: otp/send.go [hunk 6]
```diff
@@ -0,0 +1,63 @@
+package otp
+
+import (
+	"encoding/json"
+	"fmt"
```
Comment (UmanShahzad) [inline otp/send.go:5]:
```suggestion
	"encoding/json"
```

**Review by UmanShahzad — APPROVED** [review]

**Comment by UmanShahzad** [comment]
@ahmd-alli have pushed 1.1.0 - can be used now internally.

**Comment by ahmd-alli** [comment]
> @ahmd-alli have pushed 1.1.0 - can be used now internally.

Noted. Thanks.

**Pull request merged**